pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -102,10 +102,12 @@ def clean(self):
     `clean_level` is None.
 
     """
-    if self.tag == "vtec":
-        logger.info("".join(["Data provided at a clean level, further ",
-                             "cleaning may be performed using the ",
-                             "measurement error 'dtec'"]))
+    if self.tag in ["vtec", "site"]:
+        msg = "Data provided at a clean level"
+        if self.tag == "vtec":
+            msg = "".join([msg, ", further cleaning may be performed using ",
+                           "the measurement error 'dtec'"])
+        logger.info(msg)
```

Comment:
brainstorming idea to keep repeated log entries to a minimum. What about setting self.clean_level = None, with a note, in this function so the message is only logged once?

Reply:
That goes against what we've been doing as our standard, so should be brought up at the development meeting for discussion and implementation across all ecosystem packages.